plugin-libs/plugin-model/src/test/java/com/salesforce/bazel/eclipse/model/BazelOutputParserTest.java

```diff
@@ -66,7 +67,8 @@ public void testMultipleJavaErrors() {
             "sayhello/src/main/java/com/blah/foo/hello/Main.java:16: error: cannot find symbol", "    blah 1 2 3",
             "             ^",
             "ERROR: /Users/stoens/bazel-build-example-for-eclipse/sayhello/BUILD:1:1: Building sayhello/libsayhello.jar (2 source files) failed (Exit 1)",
-            "sayhello/src/main/java/com/blah/foo/hello/Main.java:17: error: cannot find symbols");
+            "sayhello/src/main/java/com/blah/foo/hello/Main.java:17: error: cannot find symbols",
+            "INFO: Elapsed time: 0.196s, Critical Path: 0.03s");
 
         List<BazelBuildError> errors = p.getErrorBazelMarkerDetails(lines);
 
```

Comment:
Are there no additional test cases to add here, based on your changes?

Reply:
I'll add some additional test cases. Because the method for multiline calls the method that consumes single line, I thought this could also cover the method for parsing single line.

Reply:
Aside from this, is it valid to assume that Java errors will always be composed of three parts: Error status line, error source line, error details line? For example, `ERROR: /Users/stoens/bazel-demo/main_usecases/java/simplejava-mvnimport/projects/libs/banana/banana-api/BUILD:1:1: Building libbanana-api.jar (2 source files) failed (Exit 1)` would be status line, `projects/libs/banana/banana-api/src/main/java/demo/banana/api/Banana.java:50: error: cannot find symbol` would be error source line, `    this.numSeeds = numSeeds;` would be details line. I tested some cases and they all seem to follow this rule. (Although one error status line can be followed by multiple error source + details).

Reply:
We don't know for sure all possible formatting of the errors - this is purely experimental, by looking at the bazel output.  Of course we could also dig into the bazel (javac?) source code, but that doesn't seem necessary...at least not yet.  All errors we have seen do seem to follow that format you describe.  

Do we need to fix the parsing to support error status + multiple error source + details?  You could add a new issue for that ...

Reply:
I looked at the code, I think it can handle that case. I will add a test case for this. However, that case details would be empty except for last error?